src/test/java/spoon/test/template/testclasses/InvocationTemplate.java

```diff
@@ -15,15 +15,15 @@ void invoke() {
 	
 	@Local
 	public InvocationTemplate(CtTypeReference<?> ifaceType, String methodName) {
-		this.ifaceType = ifaceType;
-		this.methodName = methodName;
+		this._IFace = ifaceType.getSimpleName();

```

Comment:
This is wrong, because ifaceType usually comes from different package then template and the package is lost at this place. You cannot replace parameters with type CtTypeReference by type String.